sympy/assumptions/refine.py

```diff
@@ -336,6 +336,29 @@ def refine_sign(expr, assumptions):
     return expr
 
 
+def refine_matrixelement(expr, assumptions):
+    """
+    Handler for symmetric part
+
+    Examples
+    ========
+
+    >>> from sympy.assumptions.refine import refine_matrixelement
+    >>> from sympy import Q
+    >>> from sympy.matrices.expressions.matexpr MatrixSymbol
+    >>> X = MatrixSymbol('X', 3, 3)
+    >>> refine_matrixelement(X[0, 1], Q.symmetric(X))
+    X[0, 1]
+    >>> refine_matrixelement(X[1, 0], Q.symmetric(X))
+    X[0, 1]
+    """
+    from sympy.matrices.expressions.matexpr import MatrixElement
+    arg = expr.args[0]
+    if ask(Q.symmetric(arg), assumptions):
+        if (expr.args[1] > expr.args[2]):
```

Comment:
```suggestion
        if expr.args[1] > expr.args[2]:
```

Reply:
This should use `ask(Q.positive(expr.args[1] - expr.args[2]))`. Never do `if x > y` in SymPy, because it will raise an error if x and y are symbolic.

Reply:
I have made the suggested changes